validator/tests/test_checks.py

```diff
@@ -230,6 +230,87 @@ def test_invalid_geo_id_national(self):
+        assert "01001" not in validator.raised_errors[0].expression
+        assert "88888" in validator.raised_errors[0].expression
+        assert "99999" in validator.raised_errors[0].expression
+
+    def test_invalid_geo_id_msa(self):
+        validator = Validator(self.params)
+        df = pd.DataFrame(["10180", "88888", "99999"], columns=["geo_id"])
+        validator.check_bad_geo_id_value(df, "name", "msa")
+
+        assert len(validator.raised_errors) == 1
+        assert "check_bad_geo_id_value" in validator.raised_errors[0].check_data_id
+        assert len(validator.raised_errors[0].expression) == 2
+        assert "10180" not in validator.raised_errors[0].expression
+        assert "88888" in validator.raised_errors[0].expression
+        assert "99999" in validator.raised_errors[0].expression
+
+    def test_invalid_geo_id_hrr(self):
+        validator = Validator(self.params)
+        df = pd.DataFrame(["1", "11", "111", "8", "88", "888"], columns=["geo_id"])
+        validator.check_bad_geo_id_value(df, "name", "hrr")
+
+        assert len(validator.raised_errors) == 1
+        assert "check_geo_id_value" in validator.raised_errors[0].check_data_id
+        assert len(validator.raised_errors[0].expression) == 3
+        assert "1" not in validator.raised_errors[0].expression
+        assert "11" not in validator.raised_errors[0].expression
+        assert "111" not in validator.raised_errors[0].expression
+        assert "8" in validator.raised_errors[0].expression
+        assert "88" in validator.raised_errors[0].expression
+        assert "888" in validator.raised_errors[0].expression
+
+    def test_invalid_geo_id_state(self):
+        validator = Validator(self.params)
+        df = pd.DataFrame(["aa", "ak"], columns=["geo_id"])
+        validator.check_bad_geo_id_value(df, "name", "state")
+
+        assert len(validator.raised_errors) == 1
+        assert "check_geo_id_value" in validator.raised_errors[0].check_data_id
+        assert len(validator.raised_errors[0].expression) == 1
+        assert "ak" not in validator.raised_errors[0].expression
+        assert "aa" in validator.raised_errors[0].expression
+
+    def test_uppercase_geo_id(self):
+        validator = Validator(self.params)
+        df = pd.DataFrame(["ak", "AK"], columns=["geo_id"])
+        validator.check_bad_geo_id_value(df, "name", "state")
+
+        assert len(validator.raised_errors) == 0
+        assert len(validator.raised_warnings) == 1
+        assert "check_geo_id_lowercase" in validator.raised_warnings[0].check_data_id
+        assert "AK" in validator.raised_warnings[0].expression
+
+    def test_invalid_geo_id_national(self):
+        validator = Validator(self.params)
+        df = pd.DataFrame(["us", "zz"], columns=["geo_id"])
+        validator.check_bad_geo_id_value(df, "name", "national")
+
+        assert len(validator.raised_errors) == 1
+        assert "check_geo_id_value" in validator.raised_errors[0].check_data_id
```

Comment:
```suggestion
        assert "check_bad_geo_id_value" in validator.raised_errors[0].check_data_id
```

Update due to check string/method name change.